src/publicsuffix2/__init__.py

```diff
@@ -61,7 +62,7 @@
 
 class PublicSuffixList(object):
 
-    def __init__(self, psl_file=None):
+    def __init__(self, psl_file=None, idna=True):
```

Comment:
Would it be OK if we made idna=False by default to avoid breaking compat? 
Or this was always the cases and the recent updates have broken that?

Reply:
the PSL is no longer IDNA-encoded, so if the default is idna=False, the module will incorrectly process IDNA-encoded domains. I'm not sure when Mozilla changed to UTF-8 encoding; rather annoying... as it took me awhile to realize the processing broke.  so.. that's why I have it this way.... I suspect a lot of people are incorrectly processing the psl.  otoh, instead of forcing the encoding, it seemed better to allow the option.

Reply:
OK, thanks!